0-ad-history-mod-package/simulation/templates/structures/han/wallset_stone.xml

```diff
@@ -3,6 +3,12 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Yōng</SpecificName>
+    <History>
+        Over the course of history, the Chinese were often under the threat of the nomadic horsemen inhabiting the Northern steppe. In the first millennium BC, the Chinese started building walls to protect themselves. And in the Qin dynasty (predecessor of the Han dynasty) the many individual pieces were connected to form a continuous wall, the Great Wall of China.
+        However, as the Han dynasty was formed, the nomadic people's united under a people called Xiognu. Because they were frequently raiding border regions, the Han declared war in the late first century BC. As part of their strategy, they renovated the wall and extended it even further. Once they had taken control over the strategically important Hexi Corridor,  a pass through the mountains west, they also constructed incredibly long walls along it. The Han in total doubled the entire Great Wall's length, now measuring more than 6,200 miles (10,000 kilometers). Longer than at any other point in history.  
+        The wall was constructed from material's of surrounding lands. This often meant rammed earth, clay, gravel, or various plants. The materials were dried in the sun and stacked in layers to form the wall. Although this might seem very simple and primitive, it was a very durable method. While some segments have not only survived the course of time, others even though exposed to harsh desert conditions are, after more than 2,000 years, still standing today. 
```

Comment:
```suggestion
        The wall was constructed from material found in the surrounding area. This often meant rammed earth, clay, gravel, or various plants. The materials were dried in the sun and stacked in layers to form the wall. Although this might seem very simple and primitive, it was a very durable method. While some segments have not only survived the course of time, others even though exposed to harsh desert conditions are, after more than 2,000 years, still standing today. 
```